src/current/_includes/v24.2/known-limitations/generic-query-plan-limitations.md

```diff
@@ -0,0 +1,2 @@
+- Because [generic query plans]({% link {{ page.version.version }}/cost-based-optimizer.md %}#query-plan-cache) use lookup joins instead of the scans and revscans used by custom query plans, generic query plans do not perform as well as custom query plans in some cases. [#128916](https://github.com/cockroachdb/cockroach/issues/128916)
+- [Generic query plans]({% link {{ page.version.version }}/cost-based-optimizer.md %}#query-plan-type) are not included in the [plan cache]({% link {{ page.version.version }}/cost-based-optimizer.md %}#query-plan-cache). This means that each time a CockroachDB session prepares and executes a statement using a generic query plan, it must optimize, even if nothing has changed about the tables involved. To reuse generic query plans for maximum performance, a prepared statement should be executed multiple times instead of prepared and executed once. [#128911](https://github.com/cockroachdb/cockroach/issues/128911)
```

Comment:
This sentence is not quite right. It means a generic query plan built and optimized for a prepared statement in one session cannot be used by another session.

Reply:
will fix in a follow-up PR now!